packages/pluggableWidgets/signature-native/src/Signature.tsx

```diff
@@ -0,0 +1,79 @@
+import { flattenStyles } from "@native-mobile-resources/util-widgets";
+import { executeAction } from "@widgets-resources/piw-utils";
+import { createElement, ReactElement, useCallback, useRef } from "react";
+import { View, Text } from "react-native";
+import SignatureScreen, { SignatureViewRef } from "react-native-signature-canvas";
+import { extractStyles } from "../../../tools/util-widgets/src/styles";
+import { Touchable } from "../components/Touchable";
+
+import { SignatureProps } from "../typings/SignatureProps";
+import { SignatureStyle, defaultSignatureStyle, webStyles } from "./ui/Styles";
+
+export type Props = SignatureProps<SignatureStyle>;
+
+export function Signature(props: Props): ReactElement {
+    const ref = useRef<SignatureViewRef>(null);
+    const styles = flattenStyles(defaultSignatureStyle, props.style);
+    const [signatureProps, containerStyles] = extractStyles(styles.container, ["penColor", "backgroundColor"]);
+    const [buttonClearContainerProps, buttonClearContainerStyles] = extractStyles(styles.buttonClearContainer, [
+        "rippleColor",
+        "activeOpacity",
+        "underlayColor"
+    ]);
+    const [buttonSaveContainerProps, buttonSaveContainerStyles] = extractStyles(styles.buttonSaveContainer, [
+        "rippleColor",
+        "activeOpacity",
+        "underlayColor"
+    ]);
+    const buttonCaptionClear =
```

Comment:
Can we simplify with:
```suggestion
    const buttonCaptionClear = props.buttonCaptionClear?.value ?? "Clear";
```